presto-main/src/main/java/io/prestosql/operator/scalar/BitwiseFunctions.java

```diff
@@ -23,6 +23,16 @@
 
 public final class BitwiseFunctions
 {
+    private static final long TINYINT_MASK = (1L << 8) - 1;
+    private static final long TINYINT_SIGNED_BIT = 1L << 7;
```

Comment:
This form makes it easier to read the masks:

```suggestion
    private static final long TINYINT_MASK = 0b1111_1111L;
    private static final long TINYINT_SIGNED_BIT = 0b1000_0000L;
```